siac/hostcmd.go

```diff
@@ -141,11 +141,8 @@ func hostcmd() {
 
 	// convert accepting bool
 	accept := yesNo(is.AcceptingContracts)
-	// convert price to SC/TB/mo
-	price, err := modules.StoragePriceToHuman(is.MinimumStoragePrice)
-	if err != nil {
-		price = ^uint64(0)
-	}
+	// convert price from bytes/block to TB/Month
+	price := currencyUnits(is.MinimumStoragePrice.Mul(types.NewCurrency64(4320e12)))
```

Comment:
why is `StoragePriceToHuman` failing? That means that somewhere something else has bad units, changing it here is not the right action